src/modules/mc_pos_control/PositionControl/ControlMath.hpp

```diff
@@ -43,6 +43,8 @@
 #include <matrix/matrix/math.hpp>
 #include <uORB/topics/vehicle_attitude_setpoint.h>
 
+static constexpr float A_GRAVITY = 9.80665f; // m/s^2 as defined by the SI standard
```

Comment:
CONSTANTS_ONE_G

Reply:
Good reference, I didn't know. I'll put that in but it's a bit starnge that then the multicopter position control depends on the ecl/geo library...

Reply:
That's now in the rebased version.

Reply:
Which part of that do you find strange? We also depend on it for the projection functions.